mvn tf:init -DbackendConfig=BackendConfigFileAbsolutePath causes terraform init to crash

In Version 0.11, when we run following command

mvn tf:init -DbackendConfig=BackendConfigFileAbsolutePath

Then, it fails with below error because it calls terraform init with flag -backend-config='BackendConfigFileAbsolutePath' instead of -backend-config="BackendConfigFileAbsolutePath"

Using single quotes ' instead of double quotes " for value of -backend-config parameter causes terraform init to crash

This issue does not happen in Version 0,10

Error :
C:\apache-maven-3.6.3\bin\mvn tf:init -DbackendConfig=BackendConfigFileAbsolutePath 
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for ******:terraform_gcp:jar:1.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ line 167, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-surefire-plugin is missing. @ line 171, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ line 66, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-install-plugin is missing. @ line 56, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-deploy-plugin is missing. @ line 122, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ---------------< ******:terraform_gcp >---------------
[INFO] Building terraform_gcp 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> tf-maven-plugin:0.11:init (default-cli) > :get @ terraform_gcp >>>
[INFO] 
[INFO] --- tf-maven-plugin:0.11:get (get) @ terraform_gcp ---
[INFO] Getting artifact dependencies from Maven
[WARN] Maven will be executed in interactive mode, but no input stream has been configured for this MavenInvoker instance.
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for ******:terraform_gcp:jar:1.0-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ line 167, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-surefire-plugin is missing. @ line 171, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-jar-plugin is missing. @ line 66, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-install-plugin is missing. @ line 56, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-deploy-plugin is missing. @ line 122, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO] 
[INFO] ---------------< ******:terraform_gcp >---------------
[INFO] Building terraform_gcp 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.8:copy-dependencies (default-cli) @ terraform_gcp ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.512 s
[INFO] Finished at: 2020-08-29T20:05:45+05:30
[INFO] ------------------------------------------------------------------------
[INFO] Expanding artifacts from ****\.tfmodules
[INFO] 
[INFO] <<< tf-maven-plugin:0.11:init (default-cli) < :get @ terraform_gcp <<<
[INFO] 
[INFO] 
[INFO] --- tf-maven-plugin:0.11:init (default-cli) @ terraform_gcp ---
[INFO] *** directory: ****\tf\gcp ***
[INFO] *** command: terraform init -backend-config='BackendConfigFileAbsolutePath'  ***
[INFO] 2020/08/29 20:05:46 [INFO] Terraform version: 0.13.0  
[INFO] 2020/08/29 20:05:46 [INFO] Go runtime version: go1.14.2
[INFO] 2020/08/29 20:05:46 [INFO] CLI args: []string{"C:\\WINDOWS\\terraform.exe", "init", "-backend-config='BackendConfigFileAbsolutePath'"}
[INFO] 2020/08/29 20:05:46 [DEBUG] Attempting to open CLI config file: C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\terraform.rc
[INFO] 2020/08/29 20:05:46 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
[INFO] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
[INFO] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\terraform.d\plugins
[INFO] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\HashiCorp\Terraform\plugins
[INFO] 2020/08/29 20:05:46 [INFO] CLI command args: []string{"init", "-backend-config='BackendConfigFileAbsolutePath'"}
[INFO] Initializing the backend...
[INFO] panic: runtime error: invalid memory address or nil pointer dereference
[INFO] [signal 0xc0000005 code=0x0 addr=0x20 pc=0x205279c]
[INFO] goroutine 1 [running]:
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).backendConfigOverrideBody(0xc000642fc0, 0x272cc0e, 0xf, 0xc000115480, 0xc000523920, 0xc0000496b0, 0x29, 0x0, 0x0, 0x17762fa700000000)
[INFO] 	/home/circleci/project/project/command/init.go:862 +0x8bc
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).initBackend(0xc000642fc0, 0xc0000e8120, 0x272cc0e, 0xf, 0xc000115480, 0x1, 0x0, 0x0, 0x0, 0xc0005cf8c8, ...)
[INFO] 	/home/circleci/project/project/command/init.go:374 +0x4ad
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc000642fc0, 0xc000080060, 0x1, 0x2, 0xc0001d64d0)
[INFO] 	/home/circleci/project/project/command/init.go:227 +0xec5
[INFO] github.com/mitchellh/cli.(*CLI).Run(0xc0005ed180, 0xc0005ed180, 0xc0005cfd48, 0x1)
[INFO] 	/home/circleci/project/project/vendor/github.com/mitchellh/cli/cli.go:255 +0x1e1
[INFO] main.wrappedMain(0x0)
[INFO] 	/home/circleci/project/project/main.go:275 +0xc97
[INFO] main.realMain(0x0)
[INFO] 	/home/circleci/project/project/main.go:106 +0xe5
[INFO] main.main()
[INFO] 	/home/circleci/project/project/main.go:42 +0x41
[INFO] panic: runtime error: invalid memory address or nil pointer dereference
[INFO] [signal 0xc0000005 code=0x0 addr=0x20 pc=0x205279c]
[INFO] goroutine 1 [running]:
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).backendConfigOverrideBody(0xc000642fc0, 0x272cc0e, 0xf, 0xc000115480, 0xc000523920, 0xc0000496b0, 0x29, 0x0, 0x0, 0x17762fa700000000)
[INFO] 	/home/circleci/project/project/command/init.go:862 +0x8bc
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).initBackend(0xc000642fc0, 0xc0000e8120, 0x272cc0e, 0xf, 0xc000115480, 0x1, 0x0, 0x0, 0x0, 0xc0005cf8c8, ...)
[INFO] 	/home/circleci/project/project/command/init.go:374 +0x4ad
[INFO] github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc000642fc0, 0xc000080060, 0x1, 0x2, 0xc0001d64d0)
[INFO] 	/home/circleci/project/project/command/init.go:227 +0xec5
[INFO] github.com/mitchellh/cli.(*CLI).Run(0xc0005ed180, 0xc0005ed180, 0xc0005cfd48, 0x1)
[INFO] 	/home/circleci/project/project/vendor/github.com/mitchellh/cli/cli.go:255 +0x1e1
[INFO] main.wrappedMain(0x0)
[INFO] 	/home/circleci/project/project/main.go:275 +0xc97
[INFO] main.realMain(0x0)
[INFO] 	/home/circleci/project/project/main.go:106 +0xe5
[INFO] main.main()
[INFO] 	/home/circleci/project/project/main.go:42 +0x41
[INFO] !!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
[INFO] Terraform crashed! This is always indicative of a bug within Terraform.
[INFO] A crash log has been placed at "crash.log" relative to your current
[INFO] working directory. It would be immensely helpful if you could please
[INFO] report the crash with Terraform[1] so that we can fix this.
[INFO] When reporting bugs, please include your terraform version. That
[INFO] information is available on the first line of crash.log. You can also
[INFO] get it by running 'terraform --version' on the command line.
[INFO] SECURITY WARNING: the "crash.log" file that was created may contain 
[INFO] sensitive information that must be redacted before it is safe to share 
[INFO] on the issue tracker.
[INFO] [1]: https://github.com/hashicorp/terraform/issues
[INFO] !!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.545 s
[INFO] Finished at: 2020-08-29T20:05:46+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.deliveredtechnologies:tf-maven-plugin:0.11:init (default-cli) on project terraform_gcp: Execution default-cli of goal com.deliveredtechnologies:tf-maven-plugin:0.11:init failed: Unexpected exit value: 2, allowed exit values: [0], executed command [cmd.exe, /c, terraform init -backend-config='BackendConfigFileAbsolutePath' ] in directory ****\tf\gcp, output was 4482 bytes:
[ERROR] 2020/08/29 20:05:46 [INFO] Terraform version: 0.13.0  
[ERROR] 2020/08/29 20:05:46 [INFO] Go runtime version: go1.14.2
[ERROR] 2020/08/29 20:05:46 [INFO] CLI args: []string{"C:\\WINDOWS\\terraform.exe", "init", "-backend-config='BackendConfigFileAbsolutePath'"}
[ERROR] 2020/08/29 20:05:46 [DEBUG] Attempting to open CLI config file: C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\terraform.rc
[ERROR] 2020/08/29 20:05:46 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
[ERROR] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory terraform.d/plugins
[ERROR] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\terraform.d\plugins
[ERROR] 2020/08/29 20:05:46 [DEBUG] ignoring non-existing provider search directory C:\WINDOWS\system32\config\systemprofile\AppData\Roaming\HashiCorp\Terraform\plugins
[ERROR] 2020/08/29 20:05:46 [INFO] CLI command args: []string{"init", "-backend-config='BackendConfigFileAbsolutePath'"}
[ERROR] 
[ERROR] Initializing the backend...
[ERROR] panic: runtime error: invalid memory address or nil pointer dereference
[ERROR] [signal 0xc0000005 code=0x0 addr=0x20 pc=0x205279c]
[ERROR] 
[ERROR] goroutine 1 [running]:
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).backendConfigOverrideBody(0xc000642fc0, 0x272cc0e, 0xf, 0xc000115480, 0xc000523920, 0xc0000496b0, 0x29, 0x0, 0x0, 0x17762fa700000000)
[ERROR] 	/home/circleci/project/project/command/init.go:862 +0x8bc
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).initBackend(0xc000642fc0, 0xc0000e8120, 0x272cc0e, 0xf, 0xc000115480, 0x1, 0x0, 0x0, 0x0, 0xc0005cf8c8, ...)
[ERROR] 	/home/circleci/project/project/command/init.go:374 +0x4ad
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc000642fc0, 0xc000080060, 0x1, 0x2, 0xc0001d64d0)
[ERROR] 	/home/circleci/project/project/command/init.go:227 +0xec5
[ERROR] github.com/mitchellh/cli.(*CLI).Run(0xc0005ed180, 0xc0005ed180, 0xc0005cfd48, 0x1)
[ERROR] 	/home/circleci/project/project/vendor/github.com/mitchellh/cli/cli.go:255 +0x1e1
[ERROR] main.wrappedMain(0x0)
[ERROR] 	/home/circleci/project/project/main.go:275 +0xc97
[ERROR] main.realMain(0x0)
[ERROR] 	/home/circleci/project/project/main.go:106 +0xe5
[ERROR] main.main()
[ERROR] 	/home/circleci/project/project/main.go:42 +0x41
[ERROR] panic: runtime error: invalid memory address or nil pointer dereference
[ERROR] [signal 0xc0000005 code=0x0 addr=0x20 pc=0x205279c]
[ERROR] 
[ERROR] goroutine 1 [running]:
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).backendConfigOverrideBody(0xc000642fc0, 0x272cc0e, 0xf, 0xc000115480, 0xc000523920, 0xc0000496b0, 0x29, 0x0, 0x0, 0x17762fa700000000)
[ERROR] 	/home/circleci/project/project/command/init.go:862 +0x8bc
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).initBackend(0xc000642fc0, 0xc0000e8120, 0x272cc0e, 0xf, 0xc000115480, 0x1, 0x0, 0x0, 0x0, 0xc0005cf8c8, ...)
[ERROR] 	/home/circleci/project/project/command/init.go:374 +0x4ad
[ERROR] github.com/hashicorp/terraform/command.(*InitCommand).Run(0xc000642fc0, 0xc000080060, 0x1, 0x2, 0xc0001d64d0)
[ERROR] 	/home/circleci/project/project/command/init.go:227 +0xec5
[ERROR] github.com/mitchellh/cli.(*CLI).Run(0xc0005ed180, 0xc0005ed180, 0xc0005cfd48, 0x1)
[ERROR] 	/home/circleci/project/project/vendor/github.com/mitchellh/cli/cli.go:255 +0x1e1
[ERROR] main.wrappedMain(0x0)
[ERROR] 	/home/circleci/project/project/main.go:275 +0xc97
[ERROR] main.realMain(0x0)
[ERROR] 	/home/circleci/project/project/main.go:106 +0xe5
[ERROR] main.main()
[ERROR] 	/home/circleci/project/project/main.go:42 +0x41
[ERROR] 
[ERROR] 
[ERROR] 
[ERROR] !!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
[ERROR] 
[ERROR] Terraform crashed! This is always indicative of a bug within Terraform.
[ERROR] A crash log has been placed at "crash.log" relative to your current
[ERROR] working directory. It would be immensely helpful if you could please
[ERROR] report the crash with Terraform[1] so that we can fix this.
[ERROR] 
[ERROR] When reporting bugs, please include your terraform version. That
[ERROR] information is available on the first line of crash.log. You can also
[ERROR] get it by running 'terraform --version' on the command line.
[ERROR] 
[ERROR] SECURITY WARNING: the "crash.log" file that was created may contain 
[ERROR] sensitive information that must be redacted before it is safe to share 
[ERROR] on the issue tracker.
[ERROR] 
[ERROR] [1]: https://github.com/hashicorp/terraform/issues
[ERROR] 
[ERROR] !!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

--- a/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/BackendConfigOptionFormatter.java
+++ b/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/BackendConfigOptionFormatter.java
@@ -12,7 +12,7 @@ public class BackendConfigOptionFormatter implements TfOptionFormatter {
   public String convert(String format, Object value) {
     String str = "";
     if (value instanceof String) {
-      str = csvConverter("-backend-config='%s' ", (String) value);
+      str = csvConverter("-backend-config=\"%s\" ", (String) value);
     }
     return str;
   }

--- a/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/TfOptionFormatter.java
+++ b/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/TfOptionFormatter.java
@@ -46,7 +46,7 @@ public interface TfOptionFormatter {
     for (Map.Entry<String,Object> entry: map.entrySet()) {
       Object val = entry.getValue();
       try {
-        sb.append(String.format("%s '%s=%s' ", format , entry.getKey(), val instanceof String ? val : mapper.writeValueAsString(val)));
+        sb.append(String.format("%s \"%s=%s\" ", format , entry.getKey(), val instanceof String ? val : mapper.writeValueAsString(val)));
       } catch (JsonProcessingException e) {
         throw new TerraformException(e);
       }

--- a/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/TfVarsOptionFormatter.java
+++ b/tf-build-tools/tf-cmd-api/src/main/java/com/deliveredtechnologies/terraform/api/converters/TfVarsOptionFormatter.java
@@ -21,7 +21,7 @@ public class TfVarsOptionFormatter implements TfOptionFormatter {
     }
 
     if (value instanceof String) {
-      str = csvConverter("-var '%s' ", (String) value);
+      str = csvConverter("-var \"%s\" ", (String) value);
     }
     return str;
   }

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformApplyTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformApplyTest.java
@@ -68,7 +68,7 @@ public class TerraformApplyTest {
     String tfCommand = captor.getValue();
 
     Assert.assertThat(tfCommand, CoreMatchers.startsWith("terraform apply"));
-    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var 'key1=value1' -var 'key2=value2' "));
+    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var \"key1=value1\" -var \"key2=value2\" "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-var-file=test1.txt -var-file=test2.txt "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-lock-timeout=1000s "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-target=module1.module2 "));
@@ -93,7 +93,7 @@ public class TerraformApplyTest {
     String tfCommand = captor.getValue();
 
     Assert.assertThat(tfCommand, CoreMatchers.startsWith("terraform apply"));
-    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var 'key1=value1' "));
+    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var \"key1=value1\" "));
 
   }
 

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformDestroyTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformDestroyTest.java
@@ -67,7 +67,7 @@ public class TerraformDestroyTest {
     String tfCommand = captor.getValue();
 
     Assert.assertThat(tfCommand, CoreMatchers.startsWith("terraform destroy"));
-    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var 'key1=value1' -var 'key2=value2' "));
+    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var \"key1=value1\" -var \"key2=value2\" "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-var-file=test1.txt -var-file=test2.txt "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-lock-timeout=1000s "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-target=module1.module2 "));

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformInitTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformInitTest.java
@@ -80,7 +80,7 @@ public class TerraformInitTest {
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-upgrade "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-from-module=module_dir"));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-force-copy"));
-    Assert.assertThat(tfCommand, CoreMatchers.containsString("-backend-config='bucket=foo' -backend-config='key=bar' "));
+    Assert.assertThat(tfCommand, CoreMatchers.containsString("-backend-config=\"bucket=foo\" -backend-config=\"key=bar\" "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-plugin-dir=/plugins "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-get-plugins=false "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-verify-plugins=false "));

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformPlanTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/TerraformPlanTest.java
@@ -82,7 +82,7 @@ public class TerraformPlanTest {
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-refresh=false "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-state=my.tfstate "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-target=module1.module2 "));
-    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var 'key1=value1' -var 'key2=value2' "));
+    Assert.assertThat(tfCommand, CoreMatchers.containsString("-var \"key1=value1\" -var \"key2=value2\" "));
     Assert.assertThat(tfCommand, CoreMatchers.containsString("-var-file=test1.txt -var-file=test2.txt "));
   }
 

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/converters/TfCliOptionBuilderTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/converters/TfCliOptionBuilderTest.java
@@ -28,7 +28,7 @@ public class TfCliOptionBuilderTest {
     TfCliOptionBuilder tfCliOptionBuilder = new TfCliOptionBuilder(TfCliOptionBuilder.initializeFormatterRegistry());
     String options = tfCliOptionBuilder.convert(TerraformApply.Option.values(), properties);
 
-    Assert.assertThat(options, CoreMatchers.containsString("-var 'key1=value1' -var 'key2=value2' "));
+    Assert.assertThat(options, CoreMatchers.containsString("-var \"key1=value1\" -var \"key2=value2\" "));
     Assert.assertThat(options, CoreMatchers.containsString("-var-file=test1.txt -var-file=test2.txt "));
     Assert.assertThat(options, CoreMatchers.containsString("-lock-timeout=1000 "));
     Assert.assertThat(options, CoreMatchers.containsString("-target=module1.module2 "));

--- a/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/converters/TfVarsOptionFormatterTest.java
+++ b/tf-build-tools/tf-cmd-api/src/test/java/com/deliveredtechnologies/terraform/api/converters/TfVarsOptionFormatterTest.java
@@ -13,7 +13,7 @@ public class TfVarsOptionFormatterTest {
     String format = "-var";
     String value = "key1=value1";
 
-    Assert.assertEquals(formatter.convert(format, value), "-var 'key1=value1' ");
+    Assert.assertEquals(formatter.convert(format, value), "-var \"key1=value1\" ");
   }
 
   @Test
@@ -21,6 +21,6 @@ public class TfVarsOptionFormatterTest {
     String format = "-var";
     String value = "key1=value1, key2=value2,key3=value3";
 
-    Assert.assertEquals(formatter.convert(format, value), "-var 'key1=value1' -var 'key2=value2' -var 'key3=value3' ");
+    Assert.assertEquals(formatter.convert(format, value), "-var \"key1=value1\" -var \"key2=value2\" -var \"key3=value3\" ");
   }
 }

